lib/prefix_flag.c

```diff
@@ -125,10 +125,10 @@ extern const char* process_prefix_flag (const char* short_opt, int argc, char **
 		spw_setdbname(spw_db_file);
 
 #ifdef ENABLE_SUBIDS
-		xasprintf(&suid_db_file, "%s/%s", prefix, "/etc/subuid");
+		xasprintf(&suid_db_file, "%s/%s", prefix, SUBUID_FILE");
```

Comment:
You forgot to remove `"` after `SUBUID_FILE`

Reply:
ehh, so it seems. Wonder why I didn't see any mail with your comment

Reply:
fixed now